.docker/ci/Dockerfile

```diff
@@ -1,48 +1,85 @@
-# moveit/moveit:melodic-ci
+# moveit/moveit2:crystal-ci
 # Sets up a base image to use for running Continuous Integration on Travis
+FROM ros:crystal-ros-base-bionic
 
-FROM ros:melodic-ros-base
-MAINTAINER Dave Coleman dave@picknik.ai
+LABEL maintainer="mike@picknik.ai"
 
 ENV TERM xterm
 
 # Setup catkin workspace
-ENV CATKIN_WS=/root/ws_moveit
-WORKDIR $CATKIN_WS
+ENV ROS_WS=/root/ws_moveit
+ENV ROS_DISTRO crystal
+
+WORKDIR $ROS_WS
 
 # Commands are combined in single RUN statement with "apt/lists" folder removal to reduce image size
 # https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers
-RUN \
-    mkdir src && \
-    cd src && \
+
+RUN mkdir src && \
+    cd $ROS_WS/src && \
     #
     # Download moveit source, so that we can get necessary dependencies
-    wstool init --shallow . https://raw.githubusercontent.com/ros-planning/moveit/${ROS_DISTRO}-devel/moveit.rosinstall && \
+    wstool init --shallow . https://raw.githubusercontent.com/ros-planning/moveit2/moveit2-ci/moveit.rosinstall && \
```

Comment:
moveit_ci and other tools  assume the use of a rosinstall file. I am happy to adopt vcs so long as we keep the .rosinstall format for now. That said, I think that may be scope creep for this PR since it works fine as is.

Reply:
(I believe vcs works fine with .rosinstall files)

Reply:
> That said, I think that may be scope creep for this PR since it works fine as is.

Fair, we'll get em next time.